proto/backend.proto

```diff
@@ -306,3 +306,122 @@ message StreamPacket {
   // JSON-encoded data to publish into a channel.
   bytes data = 1;
 }
+
+//-----------------------------------------------------------------
+// AdmissionControl -- EXPERIMENTAL and is subject to change until 12.??
+//-----------------------------------------------------------------
+
+service AdmissionControl {
+  // Verify if a resource can be saved or return a descriptive error
+  rpc ValidateAdmission(AdmissionRequest) returns (AdmissionResponse);
+
+  // Return a modified copy of the request that can be saved or a descriptive error
+  rpc MutateAdmission(AdmissionRequest) returns (AdmissionResponse);
+
+  // Convert a resource to a new version
+  rpc ConvertObject(ConversionRequest) returns (AdmissionResponse);
+}
+
+// Identify the Object properties
+// EG, the union of: metav1.GroupVersionKind and metav1.GroupVersionResource
+message GroupVersionKindResource {
```

Comment:
@joeblubaugh -- is this too weird?  Should I stick with GVK and leave out the resource?  Resource is useful because it is the URL path 🤷🏻 

The k8s flavor has has both GVK and GVR

Reply:
Seems like a good question for @radiohead - I don't mind having them collapsed here except for the part where it deviates from the k8s model. But this internal service doesn't seem like it needs to be 100% k8s compatible.

Reply:
GVK sounds fine for me.